### PR TITLE
Proofread docs, make links more SEO-friendly

### PR DESCRIPTION
**Pull request by NickVolynkin**

### CODE REVIEW

**Comment by NickVolynkin** [comment]
@DifferentialOrange please, review this )

File: doc/monitoring/grafana_dashboard.rst [hunk 1]
```diff
@@ -6,11 +6,11 @@ Grafana dashboard
...
 `Grafana Official & community built dashboards <https://grafana.com/grafana/dashboards>`_.
-You can find version for Prometheus datasource on
-`this page <https://grafana.com/grafana/dashboards/13054>`_ and version for
-InfluxDB datasource on `this page <https://grafana.com/grafana/dashboards/12567>`_.
+There's a version
+`for Prometheus data source <https://grafana.com/grafana/dashboards/13054>`_
+and one `for InfluxDB data source <https://grafana.com/grafana/dashboards/12567>`_.
 Tarantool Grafana dashboard is a ready for import template with basic memory,
-space operations and HTTP load panels, based on default `metrics <https://github.com/tarantool/metrics>`_
+space operations, and HTTP load panels, based on default `metrics <https://github.com/tarantool/metrics>`_
```
Comment (NickVolynkin) [inline doc/monitoring/grafana_dashboard.rst:13]: > A, B, and C

typical English thing, called serial comma or Oxford comma

File: doc/monitoring/grafana_dashboard.rst [hunk 2]
```diff
@@ -71,14 +72,14 @@ metrics path as it was configured on Tarantool instances:
         metrics_path: "/metrics/prometheus"
 
 
-To collect metrics for InfluxDB, you must use Telegraf agent.
```
Comment (NickVolynkin) [inline doc/monitoring/grafana_dashboard.rst:74]: `must` if for cases when somebody can get hurt :)

File: doc/monitoring/grafana_dashboard.rst [hunk 3]
```diff
@@ -71,14 +72,14 @@ metrics path as it was configured on Tarantool instances:
...
+To collect metrics for InfluxDB, use the Telegraf agent.
 First off, configure Tarantool metrics output in ``json`` format
 with :ref:`cartridge.roles.metrics <cartridge-role>` configuration or
-corresponding :ref:`output plugin <json>`. To start collecting metrics,
+corresponding :ref:`JSON output plugin <json>`. To start collecting metrics,
 add `http input <https://github.com/influxdata/telegraf/blob/release-1.17/plugins/inputs/http/README.md>`_
 to Telegraf configuration including each Tarantool instance metrics URL:
 
-..  code-block:: text
+..  code-block:: toml
```
Comment (NickVolynkin) [inline doc/monitoring/grafana_dashboard.rst:82]: It really is, check https://raw.githubusercontent.com/influxdata/telegraf/release-1.17/plugins/inputs/http/README.md

File: doc/monitoring/grafana_dashboard.rst [hunk 4]
```diff
@@ -139,7 +140,7 @@ Set dashboard name, folder, uid (if needed), and datasource-related query parame
 Troubleshooting
 -------------------------------------------------------------------------------
 
-If no data presents on graphs, ensure that you set up datasource and job/measurement correctly.
```
Comment (NickVolynkin) [inline doc/monitoring/grafana_dashboard.rst:142]: “ensure” is more about guarantees and rule enforcement.

**Review by DifferentialOrange — APPROVED** [review]
Thank you for your contribution!

So the question is: does someone need to update tarantool/grafana-dashboard submodule commit manually in tarantool/doc after I merge this PR or it will be enough to merge it into my master branch?

**Comment by NickVolynkin** [comment]
@DifferentialOrange It should be enough just to merge it. Our CI builds update all submodule content once in 24 hours, at night; and we also can run them manually.

**Pull request merged**